pkg/buildah/util.go

```diff
@@ -40,3 +52,64 @@ func PreloadIfTarFile(images []string, transport string) ([]string, error) {
 	}
 	return ret, nil
 }
+
+func RenderTemplatesWithEnv(filePaths string, envs map[string]string) error {
```

Comment:
Can it be merged with the logic of rootfs or extract public rendering functions?

Reply:
done